using-mobile-extensions/mobile-core/configuration/README.md

```diff
@@ -1,6 +1,169 @@
 # Configuration
 
-The Configuration extension is built into the Mobile Core extension. With the extension APIs, you can generate the necessary events in the Event Hub that provide the configuration options for all installed extensions.
+The Configuration extension is built into the Mobile Core extension. It provides several different APIs for you to setup the configuration either remotely on Launch UI or locally.
+
+## Configure with the Experience Platform Launch App ID
+
+Experience Platform Launch generates a unique environment ID that the SDK uses to retrieve your configuration. This ID is generated when an app configuration is created and published to a given environment. The app is first launched and then the SDK retrieves and uses this Adobe-hosted configuration.
+
```

Comment:
You could point them to follow these steps if they don't have a mobile property setup in Launch yet: https://aep-sdks.gitbook.io/docs/getting-started/create-a-mobile-property